tools/autograd/derivatives.yaml

```diff
@@ -609,6 +609,9 @@
 - name: polygamma(int n, Tensor self) -> Tensor
   self: grad * polygamma(n + 1, self)
 
+- name: ldexp(Tensor self, Tensor other) -> Tensor
+  self: ldexp(grad, other)
+
```

Comment:
We should add some autograd tests for this. This can be done by adding some entries to https://github.com/pytorch/pytorch/blob/master/torch/testing/_internal/common_methods_invocations.py#L499, for example the following:

```
('ldexp', (S, S), (torch.randint(-10, 10, [S, S]),)),
('ldexp', (S, S), (torch.randint(-10, 10, [S]),), 'broadcast_rhs'),
('ldexp', (S), (torch.randint(-10, 10, [S, S]),), 'broadcast_lhs'),
```

Reply:
Note that if ldexp is implemented as a composite of other torch ops it will "inherit" a grad formula and won't need an explicit entry.